README.md

```diff
@@ -1,11 +1,14 @@
-# Kafka Airflow Provider
-
-
-> **Warning**
-
-> This package has been deprecated after being accepted to OSS Airflow. Please use apache-airflow[apache.kafka] instead if you're looking for a supported kafka provider. 
+## ⚠️ Discontinuation of project
+> Astronomer [donated this provider](https://github.com/apache/airflow/pull/30175) to the official Apache Airflow repository in March 2023. Since then, the original Astronomer repository and its PyPI package are no longer maintained. For more information on the new Kafka package:
```

Comment:
```suggestion
> Astronomer [donated](https://github.com/apache/airflow/pull/30175) [this provider](https://github.com/astronomer/airflow-provider-kafka) to the official Apache Airflow repository in March 2023. Since then, the original Astronomer repository and its PyPI package have been discontinued. For more information on the new Kafka package:
```